sonic-slave-bullseye/Dockerfile.j2

```diff
@@ -3,7 +3,9 @@ FROM multiarch/qemu-user-static:x86_64-arm-5.0.0-2 as qemu
 FROM multiarch/debian-debootstrap:armhf-bullseye
 COPY --from=qemu /usr/bin/qemu-arm-static /usr/bin
 {%- elif CONFIGURED_ARCH == "arm64" and MULTIARCH_QEMU_ENVIRON == "y" %}
+FROM multiarch/qemu-user-static:x86_64-aarch64-5.2.0-2 as qemu
```

Comment:
i do not know why this is needed?

Reply:
I believe it's so that if the `qemu-*-static` binaries are not loaded and persisted in memory (i.e. by using the `--persistent` flag), then there will be a version inside the container that can be used. Looks like arm has similar code as well.